### DISCUSSION TITLE
Can I manage navigation-panel's width?

### DISCUSSION THREAD

**Discussion by igel2000** [thegeeklab/hugo-geekdoc#325] [Ideas]
I have long names in navigation-panel and my site looks that
![image](https://user-images.githubusercontent.com/6433787/152675469-c59b3f3e-1489-4873-b940-eb668ae16c00.png)

Example: https://github.com/igel2000/test_hugo_geekdocs
hugo: v0.92.1

**Comment by xoxys** [comment]
No, there is no easy way to change the width. You can fully customize the default CSS by overriding the values via the `custom.css` file, but there are several places where it needs to be adjusted to work properly, especially on mobile.

Increasing the width will also become a problem on mobile devices and I think we will not officially support it.

**Comment by xoxys** [comment]
I've used the theme for an API documentation as well in the past, and I have used the page ToC for long method names instead of the sidebar, see https://corenetworks.geekdocs.de/api/corenetworks/authenticators/. This might work for you as well.

**Reply by igel2000** [reply]
Thank's for you answer!
